src/utils/keyboard/index.website.ts

```diff
@@ -0,0 +1,46 @@
+import {InteractionManager, Keyboard} from 'react-native';
+
+let isVisible = false;
+
+const isKeyboardInput = (elem: HTMLElement): boolean => {
+    const inputTypesToIgnore = ['button', 'submit', 'checkbox', 'file', 'image'];
+    return (elem.tagName === 'INPUT' && !inputTypesToIgnore.includes((elem as HTMLInputElement).type)) || elem.tagName === 'TEXTAREA' || elem.hasAttribute('contenteditable');
+};
+
+const handleFocusIn = (event: FocusEvent): void => {
+    const target = event.target as HTMLElement;
+    if (target && isKeyboardInput(target)) {
+        isVisible = true;
+    }
+};
+
+const handleFocusOut = (event: FocusEvent): void => {
+    const target = event.target as HTMLElement;
+    if (target && isKeyboardInput(target)) {
+        isVisible = false;
+    }
+};
+
+document.addEventListener('focusin', handleFocusIn);
+document.addEventListener('focusout', handleFocusOut);
+
+const dismiss = (): Promise<void> => {
+    return new Promise((resolve) => {
+        if (!isVisible) {
+            resolve();
+            return;
+        }
+
+        Keyboard.dismiss();
+        InteractionManager.runAfterInteractions(() => {
+            isVisible = false;
+            resolve();
+        });
```

Comment:
Not sure this change can detect when the keyboard is closed

Reply:
let me find another solution

Reply:
I think you can try using `visualViewport`